src/main/java/org/rcsb/cif/model/FloatColumn.java

```diff
@@ -70,4 +68,51 @@ protected String getBinaryStringData(int row) {
     public double[] getBinaryData() {
         return binaryData;
     }
+
+    /**
+     * Uses Double.MIN_VALUE for not present and Double.MAX_VALUE for unknown
+     */
+	@Override
+	public Object getUnmaskedData() {
+		int n = rowCount;
+		double[] unmasked = new double[n];
+		if (isText) {
+			for (int i = rowCount; --i >= 0;) {
+				String val = getRawTextData(i);
+				switch (val) {
+				case ".":
+				case "":
+					unmasked[i] = Double.MIN_VALUE;
```

Comment:
By the way, `Double.MIN_VALUE` is a horribly named Java constant and is the smallest double greater than 0 that can be represented using the double type. In text representation, it will be 0 due to rounding/limited precision.